base/doc/ltnews42.tex

```diff
@@ -201,10 +201,51 @@ \subsection{Requiring or testing for the Tagging support code}
 the format. To test whether the PDF management has been loaded, the test
 \cs{IfPDFManagementActiveTF} is provided.
 
+ 
+\subsection{Moving paragraph tagging into sockets}
+
+Paragraphs in \LaTeX{} can be nested, e.g., you can have a paragraph
+containing a display quote, which in turn consists of more than one
+(sub)paragraph, followed by some more text which all belongs to the
+same outer paragraph.
+
+To model such \enquote{big paragraphs} \LaTeX{} uses a structure named 
+\texttt{text-unit}\footnote{The name is under review and is likely to change in future.}
+and use \texttt{text} (rolemapped to \texttt{P}) only for (portions of) 
+the actual paragraph text\LaTeX{}. 
+
+This is semantically clear and allows processors who care to identify
+the complete paragraphs by looking for \texttt{text-unit} tags. 
+But we got also request for an option to disable the tagging of the 
+\enquote{big paragraphs}, so with this release we moved the relevant tagging 
+code into sockets. The \enquote{big paragraphs} can now be disabled by 
+assigning the \texttt{noop} plug to these sockets:
+\begin{verbatim}
+\AssignTaggingSocketPlug{para/begin/main}{noop}
+\AssignTaggingSocketPlug{para/end/main}{noop}
```

Comment:
here you have para/main but below it is para/paragraph

leftover?

or with my new suggestion para/semantic